CONTRIBUTING.md

```diff
@@ -270,6 +272,620 @@ mypy cve_bin_tool/
+| cve_bin_tool\version_signature.py                |     8 |    0 |     8 |   100% |
+| cve_bin_tool\available_fix\debian_cve_tracker.py |     6 |    3 |     3 |    50% |
+| cve_bin_tool\available_fix\redhat_cve_tracker.py |     4 |    2 |     2 |    50% |
+| cve_bin_tool\data_sources\curl_source.py         |     5 |    0 |     5 |   100% |
+| cve_bin_tool\data_sources\epss_source.py         |     6 |    1 |     5 |    83% |
+| cve_bin_tool\data_sources\gad_source.py          |     9 |    0 |     9 |   100% |
+| cve_bin_tool\data_sources\nvd_source.py          |    13 |    6 |     7 |    54% |
+| cve_bin_tool\data_sources\osv_source.py          |    11 |    4 |     7 |    64% |
+| cve_bin_tool\data_sources\redhat_source.py       |     7 |    4 |     3 |    43% |
+| cve_bin_tool\data_sources\rsd_source.py          |     6 |    3 |     3 |    50% |
+| cve_bin_tool\output_engine\console.py            |     2 |    0 |     2 |   100% |
+| cve_bin_tool\output_engine\html.py               |     4 |    0 |     4 |   100% |
+| cve_bin_tool\output_engine\pdfbuilder.py         |    18 |   17 |     1 |     6% |
+| cve_bin_tool\output_engine\print_mode.py         |     1 |    0 |     1 |   100% |
+| cve_bin_tool\output_engine\util.py               |     9 |    0 |     9 |   100% |
+| cve_bin_tool\parsers\go.py                       |     2 |    2 |     0 |     0% |
+| cve_bin_tool\parsers\java.py                     |     3 |    1 |     2 |    67% |
+| cve_bin_tool\parsers\javascript.py               |     2 |    1 |     1 |    50% |
+| cve_bin_tool\parsers\parse.py                    |     1 |    1 |     0 |     0% |
+| cve_bin_tool\parsers\perl.py                     |     2 |    2 |     0 |     0% |
+| cve_bin_tool\parsers\php.py                      |     2 |    1 |     1 |    50% |
+| cve_bin_tool\parsers\python.py                   |     4 |    3 |     1 |    25% |
+| cve_bin_tool\parsers\r.py                        |     2 |    2 |     0 |     0% |
+| cve_bin_tool\parsers\ruby.py                     |     2 |    2 |     0 |     0% |
+| cve_bin_tool\parsers\rust.py                     |     2 |    2 |     0 |     0% |
+| cve_bin_tool\parsers\swift.py                    |     2 |    2 |     0 |     0% |
+| cve_bin_tool\sbom_manager\cyclonedx_parser.py    |     4 |    1 |     3 |    75% |
+| cve_bin_tool\sbom_manager\spdx_parser.py         |     7 |    1 |     6 |    86% |
+| cve_bin_tool\sbom_manager\swid_parser.py         |     3 |    2 |     1 |    33% |
+| doc\pypi_downloads.py                            |     7 |    7 |     0 |     0% |
+| example\oot-checker\oot_checker\checker_name.py  |     1 |    1 |     0 |     0% |
+|--------------------------------------------------|-------|------|-------|--------|
+| TOTAL                                            |   415 |  119 |   296 |  71.3% |
+--------------- RESULT: PASSED (minimum: 60.0%, actual: 71.3%) ----------------
+
+```
```

Comment:
```suggestion
Interrogate shows some [report examples in their docs](https://interrogate.readthedocs.io/en/latest/#usage) or you can look at the [current cve-bin-tool reports in our linter runs on GitHub Actions](https://github.com/intel/cve-bin-tool/actions/workflows/linting.yml)
```

This was a nice thought, but it's waaaay too long (and adds too many non-words to the dictionary, as you saw), so let's just point to where people can read the live reports and see examples from the docs.

Reply:
Alternatively, if you want to give an example with just one file that shows some misses we could probably do that, but I think people will figure it out without having the example directly here.